pom.xml

```diff
@@ -1,50 +1,58 @@
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>3.0.5</version>
-		<relativePath/> <!-- lookup parent from repository -->
-	</parent>
-	<groupId>com.bootexample4</groupId>
-	<artifactId>products</artifactId>
-	<version>0.0.1-SNAPSHOT</version>
-	<name>products</name>
-	<description>Demo project for Spring Boot</description>
-	<properties>
-		<java.version>17</java.version>
-	</properties>
-	<dependencies>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-data-jpa</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-web</artifactId>
-		</dependency>
-
-		<dependency>
-			<groupId>com.h2database</groupId>
-			<artifactId>h2</artifactId>
-			<scope>runtime</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.springframework.boot</groupId>
-			<artifactId>spring-boot-starter-test</artifactId>
-			<scope>test</scope>
-		</dependency>
-	</dependencies>
-
-	<build>
-		<plugins>
-			<plugin>
-				<groupId>org.springframework.boot</groupId>
-				<artifactId>spring-boot-maven-plugin</artifactId>
-			</plugin>
-		</plugins>
-	</build>
-
-</project>
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>3.0.5</version>
+    <relativePath/>
+    <!-- lookup parent from repository -->
+  </parent>
+  <groupId>com.bootexample4</groupId>
+  <artifactId>products</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+  <name>products</name>
+  <description>Demo project for Spring Boot</description>
+  <properties>
+    <java.version>17</java.version>
+  </properties>
+  <dependencies>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-data-jpa</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.h2database</groupId>
+      <artifactId>h2</artifactId>
+      <scope>runtime</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-test</artifactId>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>io.rest-assured</groupId>
+      <artifactId>rest-assured</artifactId>
+      <version>5.4.0</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+        <groupId>org.junit.vintage</groupId>
+        <artifactId>junit-vintage-engine</artifactId>
+        <version>5.8.1</version>
+    </dependency>
```

Comment:
```
[ERROR] /C:/Program Files/Go/src/github.com/my-products-test/src/test/java/com/bootexample4/RoostTest/petPostTest.java:[16,24] cannot find symbol
[ERROR]   symbol:   class Assert
[ERROR]   location: package org.junit
```

`[ERROR] /C:/Program Files/Go/src/github.com/my-products-test/src/test/java/com/bootexample4/RoostTest/petPutTest.java:[11,22] package io.restassured does not exist`
`[ERROR] /C:/Program Files/Go/src/github.com/my-products-test/src/test/java/com/bootexample4/RoostTest/petPutTest.java:[12,27] package io.restassured.http does not exist`
`[ERROR] /C:/Program Files/Go/src/github.com/my-products-test/src/test/java/com/bootexample4/RoostTest/petPutTest.java:[13,31] package io.restassured.response does not exist`
`[ERROR] /C:/Program Files/Go/src/github.com/my-products-test/src/test/java/com/bootexample4/RoostTest/petPutTest.java:[15,29] package io.restassured does not exist`

Got these errors as the 2 dependencies were not present.

1. io.rest-assured
2. org.junit